operator.go

```diff
@@ -30,28 +32,33 @@ func main() {
 		logrus.Panic(err)
 	}
 
-	clientset, err := versioned.NewForConfig(config)
+	extClientset, err := apiextensionsclient.NewForConfig(config)
 	if err != nil {
 		logrus.Panic(err)
 	}
 
+	err = crd.MySQLClusterCreateCRD(extClientset)
+	if err != nil {
+		panic(err)
+	}
+
+	clientset, err := versioned.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
+
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	defer cancelFunc()
 
 	clusterController := controller.NewClusterController(clientset)
 	go clusterController.Run(ctx)
 
-	go logging.LogEvents(
-		ctx,
-		clusterController,
-	)
-
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 	for {
 		select {
 		case s := <-signals:
-			logrus.Infof("Received signal %#v", s)
+			logrus.Info("Received signal %#v\n", s)
```

Comment:
Does this really require an explicit new line symbol? I think it is automatically added after each call.